src/index.js

```diff
@@ -0,0 +1,3 @@
+const Client = require('./Client');
+
+exports = Client;
```

Comment:
As mentioned in other comments :point_down: 

```suggestion
import Client from './Client';

export default Client;
```

Reply:
Done!